docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -988,7 +988,7 @@ Tasks marked as failed.
 
 *Solution*:
 
-Broadly speaking the Alerting framework is meant to gracefully handle the cases where a task is failing by rescheduling a fresh run in the future. If this fails to happen, then that means something has gone wrong in the underlying implementation and this isn’t expected.
+Broadly speaking the alerting framework is meant to gracefully handle the cases where a task is failing by rescheduling a fresh run in the future. If this fails to happen, then that means something has gone wrong in the underlying implementation and this isn’t expected.
```

Comment:
This also feels like it should read `Kibana alerting`

Reply:
Updated in https://github.com/elastic/kibana/pull/119314/commits/f8dcbce022353d226332a67391e98c9d37159051